src/test/resources/FileWithoutNewline.java

```diff
@@ -0,0 +1 @@
+import java.lang.System;public class FileWithoutNewline{public static void main(String[] args){System.out.println("Hello, world!");}}
```

Comment:
Some editors, like vim, will automatically add the trailing newline, because it's generally considered a malformed file. So, some people may unintentionally add a newline to these files, and it's not that easy to strip them off in some editors. Also, I noticed you had to exclude these test cases from the `maven-rat-plugin`.

I wonder if both these problems could be resolved by simply generating the files dynamically, as part of the test case.

Reply:
Yes, I can remove the files and generate them as strings in the tests themselves. I had thought about this actually, it's just that it's a bit harder to edit that way.

Reply:
> just that it's a bit harder to edit that way.

Yeah, I know. :crying_cat_face: Java should have multi-line strings soon, though, to make stuff like this easier. :smiley_cat: